src/app/dataset-view/additional-components/flows-component/components/tile-base-widget/tile-base-widget.helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { FlowOutcome, FlowStatus, FlowSummaryDataFragment } from "src/app/api/kamu.graphql.interface";
+
+export class TileBaseWidgetHelpers {
+    public static tileWidgetClass(node: FlowSummaryDataFragment): string {
+        switch (node.status) {
+            case FlowStatus.Finished: {
+                return node.outcome ? flowOutcomeMapperClass[node.outcome] : "";
+            }
+            case FlowStatus.Queued: {
+                return "queued";
+            }
+            case FlowStatus.Running: {
+                return "running";
+            }
+            case FlowStatus.Waiting: {
+                return "waiting";
+            }
+            case FlowStatus.Scheduled: {
+                return "scheduled";
+            }
+            default:
+                return "";
+        }
+    }
+}
+
+const flowOutcomeMapperClass: Record<FlowOutcome, string> = {
+    [FlowOutcome.Success]: "success",
+    [FlowOutcome.Failed]: "failed",
+    [FlowOutcome.Aborted]: "aborted",
+    [FlowOutcome.Cancelled]: "cancelled",
+};
```

Comment:
Duplicated with the code in `DatasetFlowTableHelpers`.

Reply:
In this case, this is a class, and in another case, this is a text message. Changed the name of classes